configure.ac

```diff
@@ -501,6 +501,34 @@ AM_CONDITIONAL([OPENTHREAD_ENABLE_NCP_MTD],     [test "${enable_ncp_app_mtd}" ==
 AM_CONDITIONAL([OPENTHREAD_ENABLE_NCP_FTD],     [test "${enable_ncp_app_ftd}" == "yes"])
 AM_CONDITIONAL([OPENTHREAD_ENABLE_NCP_RADIO],   [test "${enable_ncp_app_radio}" == "yes"])
 
+#
+# Posix Radio
+#
+
+AC_ARG_WITH(
+    [posix-radio],
+    [AS_HELP_STRING([--with-posix-radio],[Specify the POSIX radio (sim|ncp) @<:@default=sim@:>@.])],
+    [
+        case "${with_posix_radio}" in
+        sim)
+            OPENTHREAD_ENABLE_POSIX_RADIO_NCP=0
```

Comment:
What about to create two defines:

OPENTHREAD_ENABLE_POSIX_RADIO_SIM
OPENTHREAD_ENABLE_POSIX_RADIO_NCP

As [here](https://github.com/bukepo/openthread/blob/157ba1cfda2ef9d00c80463a8428eec978d41c5e/configure.ac#L542).

Reply:
Your proposal is exactly what I did at first, however, that requires updating build files for windows. So I adopted the way to configure virtual time.